components/PageLink.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { Button } from '@dts-stn/service-canada-design-system'
+
+export default function PageLink(props) {
+  const router = useRouter()
+  return (
+    <>
+      <div className="pt-2 md:pt-4 my-8 border-t border-gray-light ">
+        <h2 className="font-display font-bold text-[32px] md:text-[36px]">
+          {props.lookingForText}
+        </h2>
+        <div className="font-body mb-10  text-xl">
+          <span className=" text-gray-darker" id={`link-for-${props.linkText}`}>
+            {props.accessText}{' '}
+          </span>
+          <Link href={props.href}>
+            <a className="text-blue-default hover:text-blue-hover visited:text-purple-medium underline">
+              {props.linkText}
+            </a>
+          </Link>
+        </div>
+
+        <Button
+          id={props.buttonId}
+          onClick={() => router.push('/home')}
```

Comment:
Both links on this component (button and link) would be to internal pages, and we want to ensure we use proper next Routing for that... I believe using a button with an onClick like this isn't an accessible way of routing. I believe it should be a link, styled like a button.